lumigator/jobs/inference/inference_config.py

```diff
@@ -167,6 +166,6 @@ class InferenceJobConfig(BaseInferenceJobConfig):
     job: JobConfig
     system_prompt: str | None = Field(title="System Prompt", default=None, exclude=True)
     inference_server: InferenceServerConfig | None = None
-    params: GenerationConfig | None = None
+    generation_config: GenerationConfig | None = None
```

Comment:
Is this something that you missed in the previous PR?

Reply:
yes 🙈 . But it didn't make a difference because the InferenceJobConfig in inference/schema.py did have `generation_config` set already, and since this class inherits that class, this object did end up having the parameter name as expected.

So, the byproduct of me missing this was that the InferenceJobConfig object used in the inference job had both a generation_config and a params member, but the params member was unused.